### ISSUE TITLE
-p feature has been end of life'd.

### ISSUE THREAD

**Issue by iviewadmirer** [vadmium/python-iview#21]
server:iview user$ iview-cli -p
Sorry, iview is no longer available on this device:
	Sorry, iview is no longer available on this device	(_video/endofservice_audio_650000.mp4)

**Comment by iviewadmirer** [comment]
iview-cli -a
iView auth data:
	Streaming Host: AkamaiHDUnmetered
	RTMP Token: daEbqb5bSa5aRaycCa0b1dLbJbvd0aYaecx-bvK1wi-8-pls_vEynI&aifp=v001
	HDS Token: st=1435719048~exp=1435729047~acl=/*~hmac=6afc76b2261147b40be22f63bb81af945583027f825214e535df422ed8c82d1b
	Server URL: http://iviewum-vh.akamaihd.net/z/
	Playpath Prefix: playback/_definst_/
	Unmetered: True

**Comment by jhmos** [comment]
I get the same problem unfortunately.

**Comment by matpalm** [comment]
`--series` fails with same problem..

```
[{'items': [{'category': ' ', 'description': "ABC iview advises it is no longer able 
to support streaming on this device due to limited use and/or the device's
inability to support iviews technology updates. More information is available at:
(iview.abc.net.au/support)", 'title': 'Sorry, iview is no longer available on this
device', 'url': '_video/endofservice_audio_650000.mp4', 'expires': 
datetime.datetime(2015, 9, 1, 6, 45),  'thumb': 'http://cdn.iview.abc.net.au/thumbs/i/388/IV1465H001S005590ad322d8639.83370445_1280.jpg',
'duration': 60, 'livestream': '', 'date': datetime.datetime(2015, 7, 1, 6, 45),
'home': 'http://iview.abc.net.au/programs/sorry-iview-is-no-longer-available-on-this device/IV1465H001S00', 'id': '105806692', 'size': 5000000.0}], 
'id': '105806692', 'title': 'Sorry, iview is no longer available on this device'}]
```

**Comment by matpalm** [comment]
i guess it was this rollout?
http://iview.abc.net.au/support/announcements#1-Jul-2015

**Comment by vadmium** [comment]
Thanks for pointing out the announcement link. I made a very simple fix in revision caf7267, so pulling the master branch should get it working again for you.

**Comment by sramsay64** [comment]
Thank you for the fix. Not having this working would be a big annoyance to my TV watching. So thanks for keeping the code in working order.
